documentation/docs/14-configuration.md

```diff
@@ -138,6 +139,12 @@ When pages are prerendered, the CSP header is added via a `<meta http-equiv>` ta
 
 > When `mode` is `'auto'`, SvelteKit will use nonces for dynamically rendered pages and hashes for prerendered pages. Using nonces with prerendered pages is insecure and therefore forbidden.
 
+# link_header
```

Comment:
this should be `linkHeader` since it's exposed externally. we use camelCase for external APIs and snake_case for internal APIs